my-dsl.md

```diff
@@ -2,37 +2,55 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Asymptote](http://asymptote.sourceforge.net/)
 
 # Domain
 _Describe the language's domain in five words._
 
+Creating numerical vector graphics images.
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+Evaluation is similar to most imperative languages like C.
+However, the final data that is being constructed is a "picture" object,
+which is eventually converted into an image.
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+Asymptote is definitely a general-purpose language in power. However, it is used in a very specific way
+to create vector graphics images. It is essentially unsuitable for other things that a true
+programming language would be used for, although it could technically accomplish them.
```

Comment:
I guess this leads back to today's discussion. If it's got the ability to do everything a general-purpose language can, but it would be very difficult to do things that it wasn't designed to do, does that count as sufficiently narrowing the domain?